zk-token-sdk/src/encryption/auth_encryption.rs

```diff
@@ -50,6 +50,8 @@ pub enum AuthenticatedEncryptionError {
     DerivationMethodNotSupported,
     #[error("seed length too short for derivation")]
     SeedLengthTooShort,
+    #[error("seed length too long for derivation")]
+    SeedLengthTooLarge,
```

Comment:
nit: since you use "long" everywhere else, let's make the error say that too
```suggestion
    SeedLengthTooLong,
```

Reply:
Okay, good idea!